testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/policy/PasswordPolicyTest.java

```diff
@@ -180,6 +180,8 @@ public void testBlacklistPasswordPolicyWithTestBlacklist() throws Exception {
             Assert.assertEquals(BlacklistPasswordPolicyProvider.ERROR_MESSAGE, policyManager.validate("jdoe", "blacklisted1").getMessage());
             Assert.assertEquals(BlacklistPasswordPolicyProvider.ERROR_MESSAGE, policyManager.validate("jdoe", "blacklisted2").getMessage());
             Assert.assertEquals(BlacklistPasswordPolicyProvider.ERROR_MESSAGE, policyManager.validate("jdoe", "bLaCkLiSteD2").getMessage());
+            // Passwords with upper-case present in blacklist must be found and blocked:
+            Assert.assertEquals(BlacklistPasswordPolicyProvider.ERROR_MESSAGE, policyManager.validate("jdoe", "BlackListed3").getMessage());
```

Comment:
Would be good to also test with some casing that differs from what is added to test-password-blacklist.txt to check that all verification is indeed case insensitive + any upper-case is lower-cased when loading the file